xdp-dump/xdpdump.c

```diff
@@ -648,6 +655,11 @@ static bool capture_on_interface(struct dumpopt *cfg)
 		}
 	}
 
+	/* No more error conditions, display some capture information */
+	fprintf(stderr, "listening on %s, ingress XDP program %s, "
+		"capture size %d bytes\n", cfg->iface.ifname, prog_name,
+		cfg->snaplen);
```

Comment:
", capture mode entry,exit" ?

Reply:
Lazy me, forgot! I added it to the new commit and also updated the documentation to show it.